src/components/PlantsV2/index.tsx

```diff
@@ -55,16 +55,15 @@ export default function PlantsDashboardV2(): JSX.Element {
   );
 
   useEffect(() => {
-    dispatch(requestPlantingSites(org.selectedOrganization.id));
```

Comment:
planting sites are now dispatched at the root app level and available to select from within any component